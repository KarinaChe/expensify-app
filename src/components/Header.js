import React from 'react';
import {NavLink } from 'react-router-dom';
const Header = () =>(
    <header>
      <h1>Expensify</h1>
      <NavLink to="/" className={({isActive})=>(isActive?'active':'inactive')}>Dashboard</NavLink>
      <NavLink to="/create" className={({isActive})=>(isActive?'active':'inactive')}>Create</NavLink>
      <NavLink to="/edit" className={({isActive})=>(isActive?'active':'inactive')}>Edit</NavLink>
      <NavLink to="/help" className={({isActive})=>(isActive?'active':'inactive')}>Help</NavLink>
    </header>
)

export default Header;