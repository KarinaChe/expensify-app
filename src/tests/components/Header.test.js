
import React from 'react';
import toJson from 'enzyme-to-json';
import Header from '../../components/Header';
import { shallow } from 'enzyme'



test('should renderer Header correctly',()=>{
    const wrapper = shallow(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot();
    //expect(wrapper.find('h1').text()).toBe('Expensify')
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
})
