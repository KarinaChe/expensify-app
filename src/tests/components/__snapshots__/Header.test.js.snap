// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renderer Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    className={[Function]}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    className={[Function]}
    to="/create"
  >
    Create
  </NavLink>
  <NavLink
    className={[Function]}
    to="/edit"
  >
    Edit
  </NavLink>
  <NavLink
    className={[Function]}
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
